src/redux/actions/posts.js

```diff
@@ -25,3 +25,11 @@ export const getPosts = () => dispatch => {
     },
   )
 }
+
+export const addNewPost = (link, title, postId) => {
+  return database().ref(`posts/${postId}`).set({
```

Comment:
I think you can use `push` instead of `set` to get it to generate an id for you. (Take a look at the docs!)